app/redux/reducers/SetsReducer.js

```diff
@@ -217,6 +232,20 @@ const saveHistorySet = (state, action) => {
     return Object.assign({}, state, stateChanges);
 };
 
+// DELETE_HISTORY_SET
+
+const deleteHistorySet = (state, action) => {
+    return {
+        ...state,
+        historyData: Object.keys(state.historyData).reduce((result, setID) => {
+            if (setID !== action.setID) {
+                result[setID] = state.historyData[setID];
```

Comment:
How would you confirm this is workable without UI? Remember that in redux, if you don't properly create a copy of an object, it'll still "work" it'll just act weird in weird situations. I'm totally down for alternative ways to code something syntactically, but this is I'm not certain of.

https://redux.js.org/docs/faq/Performance.html#performance-clone-state

> you do need to create a copied and updated object for each level of nesting that is affected. 

- [ ] Define how you'd confirm this outside UI as this is a fundamental redux question. Or find documentation stating that this works.

Reply:
I'm trying to remove it immutably. We are using the spread operator here so we're not mutating the state. .reduce is creating a NEW object in this case.
https://stackoverflow.com/questions/34401098/remove-a-property-in-an-object-immutably

Reply:
Nod nod, I know reduce makes a new object. Is it okay for each set though? Legitimately unsure. I realize now that I did that when I moved sets from Workout to History though hmmmm.

Reply:
Either way it's probably fine, gonna merge for now and if we notice weird things happening after delete (like you can't explain it), can look into this reducer as a potential issue.